### PR TITLE
Adding the AIR version of the weather stencil to the AIR repo

### PR DESCRIPTION
**Pull request by eddierichter-amd**
Adding AIR versions of the 1 and 4 bblock designs to example directory.

### CODE REVIEW

File: examples/weather_stencil_air/hdiff_base.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import numpy as np
```
Comment (jgmelber) [inline examples/weather_stencil_air/hdiff_base.py:1]: License header?

**Review by jgmelber — CHANGES_REQUESTED** [review]
I think we need to also have the AMD license header on these files?

**Comment by fifield** [comment]
Maybe I missed something, but I did not see any AIR design here?  I only found mlir-aie dialect...

**Comment by jgmelber** [comment]
It's just the airrt data movement version of the stencil design using HSA packets

**Pull request closed without merging**